src/main/java/no/difi/sdp/client2/asice/ArchivedASiCE.java

```diff
@@ -3,12 +3,16 @@
 public class ArchivedASiCE {
 
     private final byte[] bytes;
+    private long unzippedContentBytesCount;
 
-    public ArchivedASiCE(byte[] bytes) {
+    public ArchivedASiCE(byte[] bytes, long unzippedContentBytesCount) {
```

Comment:
Kan denne contructoren gjøres "package private", altså uten access modifier `public`? Benyttes den av noen andre enn `CreateASiCE` som ligger i samme pakke? Grunnen til at jeg gjerne skulle begrenset tilgangen til constructoren er at "correctnessen" for parameterne er veldig avhengig av at _noe annet_ har telt opp rett antall uncompressed bytes i.f.m. en zip-operasjon.